ledger-tool/src/output.rs

```diff
@@ -939,6 +939,29 @@ impl serde::Serialize for AccountsScanner {
     }
 }
 
+#[derive(Serialize, Deserialize)]
+pub struct CliAccounts {
+    pub accounts: Vec<CliAccount>,
+}
+
+impl fmt::Display for CliAccounts {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        for account in &self.accounts {
+            write!(f, "{account}")?;
+            let account_data = account.keyed_account.account.data.decode();
+            if let Some(data) = account_data {
+                if !data.is_empty() {
+                    use pretty_hex::*;
```

Comment:
I'm assuming this is just copying the existing template used elsewhere, but any idea why we put the use inside the function here and import everything? Might be nice to move it to the rest of the includes and reduce scope to `pretty_hex::PrettyHex`